apps/internal/base/base.go

```diff
@@ -279,12 +283,12 @@ func (b Client) AuthCodeURL(ctx context.Context, clientID, redirectURI string, s
 	return baseURL.String(), nil
 }
 
-func (b Client) AcquireTokenSilent(ctx context.Context, silent AcquireTokenSilentParameters) (AuthResult, error) {
+func (b Client) AcquireTokenSilent(ctx context.Context, silent AcquireTokenSilentParameters) (ar AuthResult, err error) {
```

Comment:
I'm seeing a lot of places where we added named returns, but we don't need them for a defer statement. 

The original Go team frowned on any use of named returns if defer wasn't involved.  Pike said they should never have made them.  Naked returns, where you say `return` on named returns was verboten inside Google, as it was considered harmful.  There are Go2 requests to remove them, though we all know there will never be a Go2.

I've seen a use or two of named returns outside this that were ok, but I struggle to remember what they were.